components/automate-chef-io/README.md

```diff
@@ -81,11 +88,314 @@ This will create the file and fill in the necessary frontmatter automatically.
 
 Hugo allows us to nest our data directory structure as much as necessary. You can add as many folders as necessary under `components/automate-chef-io/data/docs`.
 
-```
+```output
 .
 ├── data
 │   ├── docs
 │   │   ├── cli
 |   │   │   ├── command_one.yml
 |   │   │   └── command_two.yml
 ```
+
+## API Documentation
+
+To view the API documentation locally, 1. From the `components/automate-chef-io` directory, run the following command to start the development server.
```

Comment:
It looks like this is halfway converted from a numbered list to prose?

Reply:
```suggestion
To view the API documentation locally, navigate to the `components/automate-chef-io` directory, run the following command to start the development server:
```